app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+from app.errors import (
+    OutdatedVaccineError,
+    NotVaccinatedError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError
+        if "wearing_a_mask" in visitor and visitor["wearing_a_mask"] is False:
```

Comment:
Use the `get` method for dictionary access can be a better approach. It's a good practice for handling cases where the presence of a key is uncertain.